src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/PiglinBarterDrop.java

```diff
@@ -0,0 +1,39 @@
+package io.github.thebusybiscuit.slimefun4.core.attributes;
+
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityDeathEvent;
+import org.bukkit.event.entity.EntityDropItemEvent;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.misc.BasicCircuitBoard;
+import io.github.thebusybiscuit.slimefun4.implementation.listeners.MobDropListener;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+
+/**
+ * This interface, when attached to a {@link SlimefunItem}, provides a variable (0-100%) chance for
+ * a {@link SlimefunItem} to be dropped by a piglin on {@link EntityItemDropEvent}. 
+ * 
+ * @author dNiym
+ *
+ * @see PiglinListener
+ * 
+ */
+@FunctionalInterface
+public interface PiglinBarterDrop extends ItemAttribute {
+
+    /**
+     * Implement this method to make this SlimefunItem have a variable chance of being
```

Comment:
You could mark SlimefunItem here as a link too.

Reply:
done